samples_project/Assets/SampleViewer/Scripts/InfoMenuController.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InfoMenuController : MonoBehaviour
+{
+    private Animator animator;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        animator = GameObject.Find("InfoMenu").GetComponent<Animator>();
+
+        StartCoroutine(SlideNotification());
+    }
+
+    // Delay pop-up notification
+    private IEnumerator SlideNotification()
+    {
+        //Wait for 2 secs.
+        yield return new WaitForSeconds(2);
```

Comment:
Since all this coroutine does is wait for 2 seconds before executing code, it would be simpler to utilize the "Invoke" function which allows you to run a function after x seconds, so it would be Invoke("SlideNotification", 2).
An even better approach would be: since WaitForSeconds is frame rate dependent, we instead utilize a loop using Time.FixedDeltaTime, however this would be a small chunk of code for a slight animation and might not be worth the trouble